.github/workflows/fuzz-nightly.yml

```diff
@@ -60,7 +60,7 @@ jobs:
 
       - name: Set crashers count
         working-directory: test/fuzz
-        run: echo "::set-output name=count::$(find . -type d -name 'crashers' | xargs -I % sh -c 'ls % | wc -l' | awk '{total += $1} END {print total}')"
+        run: echo "{count}={$(find . -type d -name 'crashers' | xargs -I % sh -c 'ls % | wc -l' | awk '{total += $1} END {print total}')}"
```

Comment:
Why not add this part to $GITHUB_OUTPUT?

Reply:
It's a bug, thanks